packages/next/src/server/app-render/types.ts

```diff
@@ -269,6 +270,11 @@ export interface RenderOptsPartial {
    * statically generated.
    */
   doNotThrowOnEmptyStaticShell?: boolean
+
+  /**
+   * next config experimental.devtoolSegmentExplorer
+   */
+  devtoolSegmentExplorer?: boolean
```

Comment:
nit: i'd be ok just calling this flag segmentExplorer to trim the verbosity a bit

Reply:
When I added it, I was thinking about either giving a common prefix or grouping them into a devtool object. Maybe the 2nd one is better if we'll have more experimental feature comming later?